includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -2266,7 +2296,10 @@ private function process_css_declaration_block( RuleSet $ruleset, CSSList $css_l
 		$results = [];
 
 		if ( $ruleset instanceof DeclarationBlock ) {
-			$this->ampify_ruleset_selectors( $ruleset );
+			$results = array_merge(
+				$results,
+				$this->ampify_ruleset_selectors( $ruleset )
+			);
```

Comment:
`$results` would always be an empty array. no?

```suggestion
			$results = $this->ampify_ruleset_selectors( $ruleset );
```

Reply:
Yes, if this `if` condition remains the first one to mutate `$results`. The idea behind merging is that it's how the logic below amends `$results`.